options_test.go

```diff
@@ -0,0 +1,74 @@
+package envbuilder_test
+
+import (
+	"testing"
+
+	"github.com/coder/envbuilder"
+	"github.com/stretchr/testify/require"
+)
+
+func TestOptionsFromEnv(t *testing.T) {
```

Comment:
We'd probably want a separate test to validate the defaults with an empty env.

Reply:
Now, env variables are handled by `coder/serpent` so I don't see a need for us to test it anymore.